### DISCUSSION TITLE
Game Mode "demo" should be renamed

### DISCUSSION THREAD

**Discussion by JohnAZoidberg** [redeclipse/base#1161] [Ideas and Feature Requests]
When I first started the game I thought "demo" sounds nice and introductive as a first game. But it didn't work - because like the description says: It's only for replaying old online games.

I think it should be renamed to "replay" or something clearer to avoid confusion.

**Comment by qreeves** [comment]
I think it'd be better to have a different menu for playing back demos, and just omitting it from the mode selection menu. Feel free to discuss though.

**Reply by dc-redeclipse** [reply]
I noticed something... its hard for new players to record demos because there is no UI config for that in 2.1. Maybe this is a part of the confusion by someone

**Comment by IceflowRE** [comment]
Since when is demo an introduction? I thought that's name is tutorial.
And the word demo in a replay context is widely used in games.

Aside of that a renaming to `replay` seems to be more clear for people with less gaming background.

**Comment by cloudziq** [comment]
for me (old player) , the demos are short game releases made specifically for demonstration purposes, before internet era.  I like change to 'replay' or overall - as Quin's idea - separate menu
